images/peer/README.md

```diff
@@ -0,0 +1,146 @@
+# Quick reference
+
+-	**Where to get Docker help**:  
+	[the Docker Community Forums](https://forums.docker.com/), [the Docker Community Slack](http://dockr.ly/slack), or [Stack Overflow](https://stackoverflow.com/search?tab=newest&q=docker)
+
+-	**Where to get Hyperledger Fabric help**:  
+	[Fabric documentation](https://hyperledger-fabric.readthedocs.io/en/latest/), [Stackoverflow](https://stackoverflow.com/questions/tagged/hyperledger-fabric), or [Rocket.Chat](https://chat.hyperledger.org/channel/fabric)
```

Comment:
`Chat`, not `Rocket.Chat`

Reply:
maybe link here as well? https://wiki.hyperledger.org/display/CA/Logging+in+to+chat

Reply:
added